CHANGELOG.md

```diff
@@ -0,0 +1,349 @@
+## 1.29.0
+
+#### New Features
+
+* **Remote state files for Terraform inventory plugin**
```

Comment:
I _think_ two spaces at the end of this line will ensure the next line comes beneath it.